### ISSUE TITLE
Add error id to error mail

### ISSUE THREAD

**Issue by atifaziz** [elmah/Elmah#368]
## What new or enhanced feature are you proposing?

I would like to have an error id in the error message as well.
## What goal would this enhancement help you achieve?

Once the elmah logs an error and mails it to us, the app redirect user to a page where she can input a text with more info about how did the error happened. 

Once she does that, the info along with error id is mailed to us. We can't find the error the user writes about in the mail (e.g. search for mail with id of an error would yield two results: the elmah error and user info) and we have to go to look it up elsewhere (e.g. on server logs).

Basically I would like to have same info in both places (mail and log).

---

Originally reported on Google Code with ID 368

Reported by `havlicek.honza` on 2014-08-16 21:19:05
